source/tools/glkernel-cmd/KernelGeneration.cpp

```diff
@@ -14,6 +14,8 @@
 
 namespace
 {
+    using uint = unsigned int;
```

Comment:
It seems like this type isn't used anymore. Additionally, I advise against using this type.

Reply:
This was a compile fix for the to-be-deprecated glkernel-cmd.